jobs/integration/validation.py

```diff
@@ -2283,7 +2346,10 @@ async def nagios(model, tools):
     # 3) wait for nagios to settle
     log("waiting for nagios to settle")
     nagios_api = NagiosApi(opener.open, status_url)
-    await nagios_api.wait_for_alerts(critical=lambda c: not c, pending=lambda p: not p)
+    await nagios_api.wait_for_settle(
+        stage="after deployment",
+        timeout_insec=60 * 15,
+    )
```

Comment:
you've got 15 min to have no criticals and no pending alerts after deployment - GO!